tensorflow/compiler/aot/codegen.cc

```diff
@@ -175,7 +175,8 @@ Status GenArgMethods(const tf2xla::Config& config,
   size_t num_args = ps.parameters_size();
   // feed_size() + variable_size() is the maximum number of args as an
   // implementation may not create an argument for an unused variable.
-  if (config.feed_size() + config.variable_size() < num_args) {
+  const int num_args_int = num_args;
+  if (config.feed_size() + config.variable_size() < num_args_int) {
```

Comment:
Rather change the type of `num_args` local variable. Whenever you have a local variable that is used only once, rather than making a new copy of it we could just change its type.

Reply:
++ suggested change applied.